ui/src/app/components/webcomponents/test-step-more-action-form.component.ts

```diff
@@ -214,8 +214,20 @@ export class TestStepMoreActionFormComponent implements OnInit {
     })
   }
   getTextContent(testStep:TestStep){
-    if(testStep.isForLoop){
+    if(testStep.isForLoop) {
```

Comment:
move this to utils.
plz have a prefix variable and concatenate with contextual texting.
ex, 
`let stepText = `Step ${testStep.position + 1} - ``
stepText + 'For Loop'

Reply:
this method will be used here only , it might not be used anywhere .